source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -138,7 +138,7 @@ EmphasisParser::EmphasisState EmphasisParser::MatchText(EmphasisParser& parser,
     }
     else
     {
-        if (isEmphasisToken && parser.m_lookBehind == DelimiterType::Escape)
+        if (parser.m_lookBehind == DelimiterType::Escape)
```

Comment:
does escape char works?
looks like, these changes removing all of the escape char.